src/averageAges.js

```diff
@@ -15,11 +15,13 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const getMen = person => person.sex === 'm';
+  const getMenBornInCentury = person => person.sex === 'm'
```

Comment:
```suggestion
  const getMenBornInCentury = person => getMen(person)
```
I would reuse sex check in this codeline.

Reply:
Плюсую комент Андрія, 
isManBornInCentury 
```suggestion
  const isManBornInCentury = isMan && anotherCondition
```